classes/patreon_oauth.php

```diff
@@ -33,13 +33,18 @@ public function refresh_token($refresh_token, $redirect_uri) {
 	}
 
 	private function __update_token($params) {
+		
 		$api_endpoint = "https://api.patreon.com/oauth2/token";
-		$ch = curl_init();
-		curl_setopt($ch, CURLOPT_URL, $api_endpoint);
-		curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
-		curl_setopt($ch, CURLOPT_POST, 1);
-		curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query($params));
-		return json_decode(curl_exec($ch), true);
+	
+		$api_request = array(
+		'method' => 'POST',
```

Comment:
tab these args in

Reply:
Committed:

https://github.com/codebard/patreon-wordpress/commit/2c0b74f10b5726ce3a30c08929a8a0b26e070e67